src/devices/tuya.ts

```diff
@@ -10458,6 +10458,157 @@ const definitions: Definition[] = [
             ],
         },
     },
+    {
+        fingerprint: tuya.fingerprint('TS0601', ['_TZE204_q22avxbv']),
+        model: 'TOQCB2-80',
+        vendor: 'TuYa',
+        description: 'Smart Circuit breaker',
+        fromZigbee: [tuya.fz.datapoints],
+        toZigbee: [tuya.tz.datapoints],
+        // Important: respondToMcuVersionResponse should be false otherweise there are an avalanche of commandMcuVersionResponse messages every second.
+        // queryIntervalSeconds: is doing a pooling to update device's parameters, now define to update data every 3 minutes.
+        onEvent: tuya.onEvent({respondToMcuVersionResponse:false, queryIntervalSeconds: 3 * 60}),
+        configure: tuya.configureMagicPacket,
+        exposes: [
+            tuya.exposes.switch(), 
+            e.energy(), 
+            e.power(), 
+            e.voltage(), 
+            e.current(), 
+            e.temperature(),
+            tuya.exposes.voltageWithPhase('a'), 
+            tuya.exposes.voltageWithPhase('b'), 
+            tuya.exposes.voltageWithPhase('c'),
+            tuya.exposes.powerWithPhase('a'), 
+            tuya.exposes.powerWithPhase('b'), 
+            tuya.exposes.powerWithPhase('c'),
+            tuya.exposes.currentWithPhase('a'), 
+            tuya.exposes.currentWithPhase('b'), 
+            tuya.exposes.currentWithPhase('c'),
+            e.numeric('forward_electricity', ea.STATE).withUnit('kwh'),
+            e.enum('last_event1', ea.STATE, 
+                   ['Normal', 
+                    'Trip_Over-current', 
```

Comment:
```suggestion
                    'trip_over_current', 
```

Please make all the entries lower snake case.